apps/site/scripts/blog-data/generate.mjs

```diff
@@ -0,0 +1,11 @@
+import { writeFileSync } from 'node:fs';
+
+import generateBlogData from '../../next-data/generators/blogData.mjs';
+
+const blogData = await generateBlogData();
+
+writeFileSync(
+  new URL(`../../public/blog-data.json`, import.meta.url),
```

Comment:
Please use `node:path` instead of `new URL`

Reply:
Why?

Reply:
I can do:
```js
writeFileSync(`./public/blog-data.json`, JSON.stringify(blogData), 'utf8');
```
and that would work just fine (since the script is always run from the site's directory)

however that feels unnecessarily unclear to me.

So indeed as Aviv mentioned, I think that I can either revert this change to how I originally had it if preferred or keep the URL format, I'm happy with either 🙂

Reply:
+1 on keeping it as URL

Reply:
I was referring to using `join` but this is also fine.